APIs/EDIDAPI.raml

```diff
@@ -45,6 +45,9 @@ mediaType: application/json
           400:
             description: >
               Returned when the media profiles are rejected due to schema validation failure or if the Device doesn't support the requested mode.
+          405:
```

Comment:
`405` is sending the wrong METHOD, like a deleted.

what about https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422

"Unprocessable Entity"

```suggestion
          422:
```

Reply:
I'm not sure about that. 422 is one of the WebDAV extensions that we've previously avoided using in other NMOS APIs.
Also it's not that the request body is unprocessable as I understand it here, rather that the particular Sender - indicated in the URL path - doesn't support this operation? If so, 501 might be appropriate?

Reply:
Since we have different understanding, perhaps we are missing a phrase?

@N-Nagorny is this trying to say:

1. Return this when the Sender is not capable of applying the `media profile` 
2. This Sender does not support processing `media profile`s

Reply:
It looks for me that you're both right. I tried to handle a case when Sender doesn't support changing itself by PUTting Media Profiles.
422 looked good to me since this case makes such requests semantically erroneous but it doesn't depend on PUT content but on the method itself.
501 is maybe appropriate but avoiding WebDAV extensions breaks another case: 423 when Sender can't be modified if stream is active.

Reply:
And that means I'm wrong, because we _do_ use **423 Locked** in other specs, so that's not a reason to reject 422. So important thing is whether the semantics of 422 make sense.

Reply:
It looks for me that 501 fits better because 422 may make a user thinking that there's some right request for PUT method.

Reply:
If a Sender does not support being `PUT`ed a `media-profile`... why would it be exposed in the API?

Reply:
If a Sender does not support it, it doesn't make sense to keep it in `/senders` then. I don't know what is better: to have different lists with IS-04 or to return 501 on PUT to `/media-profiles`. I'll maybe ask in AMWA Slack.

Reply:
I specified that `/senders` and `/receivers` contain only EDID Processing compatible items and removed this code in 3619cf4c4f61df6002c10b0fcc06ce51b2ada4ec.